packages/backend/src/postHog.ts

```diff
@@ -11,9 +11,27 @@ const FLAG_CHECK_TIMEOUT = process.env.POSTHOG_CHECK_TIMEOUT
 export const postHogClient = lightdashConfig.posthog
     ? new PostHog(lightdashConfig.posthog.projectApiKey, {
           host: lightdashConfig.posthog.beApiHost,
+          fetch: async (url, options) => {
+              const response = await fetch(url, options);
+
+              if (!response.ok) {
+                  try {
+                      Logger.error('PostHog Error', response.json());
+                  } catch (e) {
+                      Logger.error('PostHog Error', response.body);
+                  }
+              }
+
+              return response;
+          },
```

Comment:
We should discuss the possibility of overwriting the request method used by posthog client
Currently they're using axios this would replace it with fetch, it probably won't make a difference, but we can log the errors ourselves

![image](https://github.com/user-attachments/assets/c9bad83a-b1e1-4464-8053-8b9417ed6d84)